src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/GoogleServicesDAO.scala

```diff
@@ -21,8 +21,6 @@ trait GoogleServicesDAO extends ReportsSubsystemStatus {
   def getBucketObjectAsInputStream(bucketName: String, objectKey: String): InputStream
   def getObjectResourceUrl(bucketName: String, objectKey: String): String
 
-  val fetchPriceList: Future[GooglePriceList]
-
```

Comment:
could you remove the `fetchPriceList` implementation from `HttpGoogleServicesDAO` as well, along with any code it obsoletes? I think you'll also be able to delete the `GoogleCloud.priceListUrl` and possibly related vals in `FireCloudConfig`.

Reply:
Done: https://github.com/broadinstitute/firecloud-orchestration/pull/1586/commits/11b680dca3d9c77096b1a8d77cf36414a05b7e52